internal/nodes/avi_model_l7_translator.go

```diff
@@ -247,10 +244,24 @@ func (o *AviObjectGraph) BuildTlsCertNode(svcLister *objects.SvcLister, tlsNode
 			utils.AviLog.Infof("key: %s, msg: key not found for secret: %s", key, secretObj.Name)
 			return false
 		}
+		altCert, ok := keycertMap[utils.K8S_TLS_SECRET_ALT_CERT]
```

Comment:
One naive doubt: are `alt.crt` and `alt.key`  section part of kubernetes secrets? I didn't get doc for these fields.